docs/design/AnoncredsW3CCompatibility.md

```diff
@@ -0,0 +1,286 @@
+# Anoncreds W3C Compatibility
+
+This design proposes to extend the Aries Cloud Agent Python (ACA-PY) to support Hyperledger AnonCreds credentials and presentations in the W3C Verifiable Credentials (VC) and Verifiable Presentations (VP) Format. The aim is to transition from the legacy AnonCreds format specified in Aries-Legacy-Method to the W3C VC format.
+<br><br>
+
+## Overview
+
+We aim to wrap the enhancements made on the Rust Framework for Anoncreds first. Then we will work on the integration of AnonCreds with W3C VC Format in ACA-PY, which includes support for issuing, verifying, and managing W3C VC Format AnonCreds credentials.
+
+Ideally the signatures will be delivered in parallel with the Javascript Framework Document.
+<br><br>
+
+## Caveats
+
+For now, we will only target compatibility with [VCDM (Verifiable Credential Data Model) 1.1](https://www.w3.org/TR/vc-data-model/) because the Rust framework we are deriving from is also working with that target. This means we would support the features being implemented in the RUST frameworks, which include:
+
+- Credentials: Verify validity of non-Creds Data Integrity proof signatures
+- Presentations: Create presentations using non-AnonCreds Data Integrity proof signature
+- Presentations: Verify validity of presentations, including non-AnonCreds Data Integrity proof signatures
+- Presentations: Support different formats (for example, DIF) of Presentation Request
+
+This may change if the Rust framework decides to support VCDM (Verifiable Credential Data Model) 2.0.
+<br><br>
+
+## Issues to consider
+
+- If and how the W3C VC Format attachments for the Issue Credential V2.0 and Present Proof V2 Aries DIDComm Protocols should be used when using AnonCreds W3C VC Format credentials.
+- How AnonCreds W3C VC Format verifiable credentials are stored by the holder such that they will be discoverable when needed for creating verifiable presentations.
+- How and when multiple signatures can/should be added to a W3C VC Format credential, enabling both AnonCreds and non-AnonCreds signatures on a single credential and their use in presentations.
```

Comment:
While I think we need to discuss/document how this should, I think it this is out of scope of this effort. I think the controller will do the operations necessary to enable this, and that ACA-Py need not orchestrate the work. At most a demo flow and documentation of any missing features is needed.